tools/c7n_gcp/tests/data/flights/node-template-query/get-compute-v1-projects-cloud-custodian-aggregated-nodeTemplates_1.json

```diff
@@ -0,0 +1,276 @@
+{
+  "body": {
+    "items": {
+      "regions/us-west2": {
+        "warning": {
+          "message": "There are no results for scope 'regions/us-west2' on this page.", 
+          "code": "NO_RESULTS_ON_PAGE", 
```

Comment:
same re stripping extraneous data with script in #4615

Reply:
done